reparo/syncer/translate.go

```diff
@@ -137,3 +144,63 @@ func genColsAndArgs(row [][]byte) (cols []string, args []interface{}, err error)
 
 	return
 }
+
+// parserSchemaTableFromDDL parse ddl query to get schema and table
+// ddl like use test; create table ....
+func parserSchemaTableFromDDL(ddlQuery string) (schema, table string, err error) {
+	stmts, _, err := parser.New().Parse(ddlQuery, "", "")
+	if err != nil {
+		return "", "", err
+	}
+
+	for _, stmt := range stmts {
```

Comment:
Is it possible that `stmts` contains more than 2 statements and involves more than 1 table?

Reply:
should not have this situation

Reply:
should we check this case and return error?

Reply:
IMHO it would be safer if we check to make sure it doesn't involve more than 1 table.

Reply:
only `drop table` and `rename table` may involve more than one table, but this seems is not a serious problem now, will fix it later